src/code/fault_ootn64.c

```diff
@@ -0,0 +1,835 @@
+                        x = 4;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0x4:
+                    if (btnPress == BTN_DDOWN) {
+                        x = 5;
+                        y = 1;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0x5:
+                    if (btnPress == BTN_DLEFT) {
+                        x = 6;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0x6:
+                    if (btnPress == BTN_CLEFT) {
+                        x = 7;
+                        y = 1;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0x7:
+                    if (btnPress == BTN_CRIGHT) {
+                        x = 8;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0x8:
+                    if (btnPress == BTN_DRIGHT) {
+                        x = 9;
+                        y = 1;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0x9:
+                    if (btnPress == (BTN_A | BTN_B)) {
+                        x = 0xA;
+                    } else if (btnPress == BTN_A) {
+                        x = 0x5B;
+                    } else if (btnPress == BTN_B) {
+                        x = 0x5C;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0x5B:
+                    if (btnPress == BTN_B) {
+                        x = 0xA;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0x5C:
+                    if (btnPress == BTN_A) {
+                        x = 0xA;
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                case 0xA:
+                    if ((btnCur == (BTN_A | BTN_B | BTN_START)) && (btnPress == BTN_START)) {
+                        f32 comboTimeS = OS_CYCLES_TO_USEC(osGetTime() - comboStartTime) / 1000000.0f;
+
+                        osSyncPrintf("入力時間 %f 秒\n", comboTimeS);
+                        if (comboTimeS <= 50.0f) {
+                            x = 0xB;
+                        } else {
+                            x = 0;
+                        }
+                    } else {
+                        x = 0;
+                    }
+                    break;
+
+                default:
+                    break;
+            }
+        }
+    }
+}
+
+extern s32 D_80105A90_unknown; // Arena_failcnt
```

Comment:
this is in __osMalloc.c .data, it could be nice to define it now rather than adding it to undefined_syms.txt. Although I'm not sure what header this declaration should go in (variables.h for now or just leave it here?)

Reply:
bump on this